keybindings/10-pop-shell-move.xml

```diff
@@ -8,6 +8,8 @@
   <KeyListEntry name="tile-move-up" description="Move window up"/>
   <KeyListEntry name="pop-workspace-down" description="Move window to lower workspace"/>
   <KeyListEntry name="pop-workspace-up" description="Move window to upper workspace"/>
+  <KeyListEntry name="pop-workspace-left" description="Move window to left workspace"/>
+  <KeyListEntry name="pop-workspace-righ" description="Move window to right workspace"/>
```

Comment:
Based on the names of other keys, this should probably be `pop-workspace-right` instead of `pop-workspace-righ`.